tests/travis.patch

```diff
@@ -1,16 +1,16 @@
 --- a/snap/snapcraft.yaml
 +++ b/snap/snapcraft.yaml
-@@ -196,6 +196,7 @@ parts:
+@@ -91,6 +91,7 @@ parts:
```

Comment:
Ouch, we accumulated a huge offset.

Reply:
Yep, good to get the patch freshened :grin: